zkevm-circuits/src/state_circuit.rs

```diff
@@ -37,23 +37,27 @@ const N_LIMBS_ACCOUNT_ADDRESS: usize = 10;
 const N_LIMBS_ID: usize = 2;
 
 /// Config for StateCircuit
-#[derive(Clone)]
-pub struct StateConfig<F: Field> {
+#[derive(Clone, Copy)]
+pub struct StateConfig {
     selector: Column<Fixed>, // Figure out why you get errors when this is Selector.
     // https://github.com/appliedzkp/zkevm-circuits/issues/407
-    rw_counter: MpiConfig<u32, N_LIMBS_RW_COUNTER>,
+    sort_keys: SortKeysConfig,
     is_write: Column<Advice>,
-    tag: Column<Advice>,
+    value: Column<Advice>,
+    lexicographic_ordering: LexicographicOrderingConfig,
+    lookups: LookupsConfig,
+    power_of_randomness: [Column<Instance>; N_BYTES_WORD - 1],
+}
+
+/// Keys for sorting the rows of the state circuit
+#[derive(Clone, Copy)]
+pub struct SortKeysConfig {
+    tag: BinaryNumberConfig<RwTableTag, 4>,
     id: MpiConfig<u32, N_LIMBS_ID>,
-    is_id_unchanged: IsZeroConfig<F>,
     address: MpiConfig<Address, N_LIMBS_ACCOUNT_ADDRESS>,
     field_tag: Column<Advice>,
```

Comment:
put `field_tag` behind `storage_key`

Reply:
why? do you want me to change the order in which the table is sorted too?

Reply:
sorry, I remember wrong after check the latest spec :).

Reply:
sorry, I remember wrong after check the latest spec :).